projects/lights_control/src/main.c

```diff
@@ -0,0 +1,14 @@
+#include <stdint.h>
+
+#include "can.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "status.h"
+#include "wait.h"
+
+#include "lights_can.h"
+#include "lights_gpio.h"
+
+int main(void) {
+  return STATUS_CODE_OK;
```

Comment:
You might as well write your event loop here so you can actually test receiving events on the actual hardware, even if you're not doing anything with events that your code raises. At the very minimum, something like

```c
while (true) {
}
```

Reply:
Yes I actually have a while loop in which I'm checking for events, that's just gonna be in my next PR. I'll add the empty while loop for now.